data/scripts/dependency-resolver.py

```diff
@@ -281,6 +281,7 @@ def handle_exec_check(args, conf, context):
     if success:
         dir_path = os.path.dirname(os.path.realpath("%s" % path))
         context.add_cond_makefile_var("%s_DIR" % dep_sym, dir_path)
+        context.add_cond_makefile_var("%s_PATH" % dep_sym, dir_path)
```

Comment:
I figured since we are using to "dir" and "path" ambiguously, we should have both _DIR and _PATH...